Live-Snap/Live-Snap/Networking/IsValidSnapchatUsernameEndpointRequest.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  IsValidSnapchatUsernameEndpointRequest.swift
+//  Live-Snap
+//
+//  Created by Baby on 3/13/18.
+//  Copyright © 2018 Oleg Abalonski. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class IsValidSnapchatUsernameEndpointRequest {
+    
+    var snapchatUsername: String
+    
+    init(snapchatUsername: String) {
+        self.snapchatUsername = snapchatUsername
+    }
+    
+    func start(completion: @escaping ((Bool) -> ())) {
+        
+        let url = "https://feelinsonice-hrd.appspot.com/web/deeplink/snapcode?username=\(snapchatUsername)&type=SVG"
+        
+        Alamofire.request(url).validate().responseString { (response: DataResponse<String>) in
+            
+            
+            switch response.result {
+            case .success:
+                
+                if let responseString = response.result.value {
+                    completion(responseString.contains(substring: "clip-path=\"url(#clip)\""))
```

Comment:
Should we implement a check for basic username guidelines? According to Snapchats Law Enforcement Guide,

Snapchat usernames:
● Must be 3-15 characters long
● Can’t contain spaces
● Must begin with a letter
● Can only contain letters, numbers, and the special characters
hyphen ( - ), underscore ( _ ), and period ( . ), EXCEPT that the username:
    ○ Can’t begin with a number, hyphen, underscore, or period
    ○ Can’t end with a hyphen, underscore, or period
● Can’t contain emojis or other symbols such as @, $, #, etc.
● Will appear only in lower-case letters within the app

-https://storage.googleapis.com/snap-inc/privacy/lawenforcement.pdf